app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -59,6 +65,10 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   private val chooserViewModel: ProfileChooserViewModel by lazy {
     getProfileChooserViewModel()
   }
+  private val wasProfileEverCreatedLiveData: LiveData<Boolean> by lazy {
+    getProfileEverCreated()
+  }
+  private var whichViewAdd: Boolean = true
```

Comment:
Remove this and pass true/false directly with an inline comment.

Similar to this line (in this ` /* attachToRoot= */ ` the inline comment):
`val binding = ProfileChooserFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)`